client/src/components/App/Content/InvestigationForm/useInvestigationForm.ts

```diff
@@ -44,9 +50,22 @@ const useInvestigationForm = (): useInvestigationFormOutcome => {
         );
 
         timeout(1900).then(() => history.push(landingPageRoute));
+        // TODO: navigate to home page + wait until swal closes
     };
 
+    const continueToNextTabByClick = () => {
+        if(currentTab.id + 1 === 3) {
+            setIsLastTab(!isLastTab);
```

Comment:
can just to true here according to the condition?

Reply:
Thought it's more "pretty" this way but ok :)